content/projects/singlecell/index.md

```diff
@@ -0,0 +1,77 @@
+ - [Matrix channel for asking for help](https://matrix.to/#/#Galaxy-Training-Network_galaxy-single-cell:gitter.im)
+
+Saying that, we'd absolutely love to have users feedback on their experiences to help us improve. You'd be very welcome to join the Community of Practice to help improve the Galaxy single cell space!
+
+# What are the goals of the single cell community of practice?
+
+🎯Goals
+- Prevent work duplication
+- Collaborate across pipeline, ensuring users feedback to developers
+- Identify community needs
+- Tool & tutorial development
+- Testing
+- Build & support user community
+
+# Who are we looking for?
+
+Anyone with energy! We love hearing from users, developers, trainers, and everyone in between who is using or developing Galaxy for single cell analysis.
+
+# How do we function?
+
+**Meetings** We meet every 6 weeks (alternating USA and AU time zones) for 1 hour.
+
+Our meeting agenda is:
+ - Random intro chatter & introductions
+ - Updates from Trello site (what people are working on / asking for help)
+    - 🐛Help/bugs
+    - Status reports & new stuff that’s happening
+ - Other cool stuff (example: Smorgasbord)
+ - Ongoing actions to send out
+
+**Emails** We send out emails to the mailing list after each meeting, at the mid-way point between meetings, and the day before meetings as a reminder.
+
+**Notes** We keep rolling notes on a googledoc and monitor our work on Trello.
+
+**Collaborative Chat Forums** When people ask for help or are working across institutes, we set up chat sub-channels to bring them together.
+
+**General Chat Forum** We have a chat forum to discuss news and ideas, and are represented in the Galaxy Community Reps chat group.
+
+<iframe src="https://calendar.google.com/calendar/embed?src=f6b4dbe4aa3d7076e740e4198aa482d6b92b52129c9f867cfb49f7ac40450796%40group.calendar.google.com&ctz=Europe%2FLondon" style="border: 0" width="800" height="600" frameborder="0" scrolling="no"></iframe>
+
+# How can you join & other important links
+
+Start by joining our [💬Matrix channel](https://matrix.to/#/#usegalaxy-eu_single-cell-workflows) and introduce yourself! (Or you can just lurk there for a bit!)
+
+ - 📪[Mailing list](https://lists.galaxyproject.org/lists/single-cell-cop.lists.galaxyproject.org/)
+ - ✅[Trello site](https://trello.com/invite/b/KLVgHvCB/ATTI53fe3f6ea71737e3aa6570e5ed148a528698CA27/sccop-tool-wrapping-tutorial-development)
+ - 🗓️[SC-CoP Google Calendar](https://calendar.google.com/calendar/embed?src=f6b4dbe4aa3d7076e740e4198aa482d6b92b52129c9f867cfb49f7ac40450796%40group.calendar.google.com&ctz=Europe%2FLondon) <-- Subscribe!
+ - 📝[Meeting Notes doc](https://docs.google.com/document/d/19W--oeFoEgfZbw9MWvky_A__554th-VG3ryOqtfmHSA/edit?usp=sharing) <--- There's a section here called 'New here?' that gives tips on how to start contributing as well!
+ - 🧑🏽‍🏫[User Matrix Channel](https://matrix.to/#/#Galaxy-Training-Network_galaxy-single-cell:gitter.im)
+
```

Comment:
A section for upcoming Galaxy single-cell events/workshops?

Reply:
Grim. If we can't automate it, we can't upkeep it.